service-registry/src/main/java/org/apache/servicecomb/serviceregistry/registry/AbstractServiceRegistry.java

```diff
@@ -183,26 +183,15 @@ public void run() {
   private void loadFrameworkVersions() {
     Framework framework = new Framework();
     framework.setName(CONFIG_FRAMEWORK_DEFAULT_NAME);
-
-    if (needSetFrameworkVersion()) {
-      framework.setVersion(FrameworkVersions.allVersions());
+    if(ServiceRegistryConfig.getMicroserviceFramework()) {
+    	framework.setVersion(FrameworkVersions.allVersions());
     }
     microservice.setFramework(framework);
     microservice.setRegisterBy(CONFIG_DEFAULT_REGISTER_BY);
   }
 
-  private boolean needSetFrameworkVersion() {
-    if (serviceCenterInfo == null) {
-      LOGGER.warn("Server startup when service center not started and cannot retrieve version info, assume latest.");
-      return true;
-    }
-
-    Version scVersion = VersionUtils.getOrCreate(serviceCenterInfo.getVersion());
-    Version frameworkVersion = VersionUtils.getOrCreate(Const.SERVICECENTER_FRAMEWORK_VERSION);
-    return scVersion.compareTo(frameworkVersion) >= 0;
-  }
 
-  private void loadStaticConfiguration() {
+private void loadStaticConfiguration() {
```

Comment:
2 space

Reply:
it's difficult to write no problem code without IDE......

Reply:
I use IDE coding, but forget format demo